config/crd/bases/volumegroup.storage.dell.com_dellcsivolumegroupsnapshots.yaml

```diff
@@ -69,6 +69,12 @@ spec:
                 items:
                   type: string
                 type: array
+              timeout:
```

Comment:
Should we update version if we're changing the CRD?

Reply:
we might have to 
we can do it in the next PR after having discussions as we need to update go.mod dependencies too